deployPreprod

```diff
@@ -1,3 +1,4 @@
 . ${NVM_DIR}/nvm.sh
 # export NODE_ENV=production
 BABEL_ENV=production nvm run `node ./bin/printNodeVersion` gulpfile.js  deployPreprod
+cp -r dist/* ../polisFileServer/prod/
```

Comment:
@urakagi Is this part of your attempt at solving the deploy problem by first doing scp deploy, then aws sync?

Reply:
Yes. The ideal solution is to make a dummy uploader and just copy files in the script, but I failed on it. I really don't need the uploaders.